src/main/java/roomescape/reservation/controller/request/ReservationCreateRequest.java

```diff
@@ -1,10 +1,14 @@
 package roomescape.reservation.controller.request;
 
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+
 import java.time.LocalDate;
 
 public record ReservationCreateRequest(
-        String name,
-        LocalDate date,
-        Long timeId
+        @NotBlank String name,
```

Comment:
Validation 애노테이션을 활용해주셨군요? 👍👍 

애노테이션 프로퍼티로 message도 지정해주면 어떨까요?😃 
```suggestion
        @NotBlank(message = "예약자명은 필수입니다.") String name,
```

Reply:
네 프로퍼티로 message도 지정해주고, ExceptionHandler에서 Validation 예외 잡아서 처리해줄 수 있도록 수정해두었습니다.😊